### ISSUE TITLE
installation problem

### ISSUE THREAD

**Issue by garfring** [agshumate/Liftoff#141]
I am getting the following error with the install commmand: 

### installation
$ module load gcc/9.3.0 openmpi/4.0.3 busco/5.2.2

Lmod is automatically replacing "intel/2020.1.217" with "gcc/9.3.0".

Due to MODULEPATH changes, the following have been reloaded:
  1) openmpi/4.0.3

$ virtualenv ~/MYENV
created virtual environment CPython3.8.10.final.0-64 in 4428ms
  creator CPython3Posix(dest=/home/age87/MYENV, clear=False, global=False)
  seeder FromAppData(download=False, pip=latest, setuptools=latest, wheel=latest, via=copy, app_data_dir=/home/age87/.local/share/virtualenv/seed-app-data/v1.0.1)
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator

$ source ~/MYENV/bin/activate

$ pip install parasail --no-index
Ignoring pip: markers 'python_version < "3"' don't match your environment
Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx512, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/parasail-1.2+computecanada-py2.py3-none-linux_x86_64.whl
Requirement already satisfied: wheel in /home/age87/MYENV/lib/python3.8/site-packages (from parasail) (0.34.2)
Requirement already satisfied: numpy in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx512/Core/scipy-stack/2021a/lib/python3.8/site-packages (from parasail) (1.21.0+computecanada)
Installing collected packages: parasail
Successfully installed parasail-1.2+computecanada

$ pip install $HOME/Liftoff-1.6.3-py3-none-any.whl
Ignoring pip: markers 'python_version < "3"' don't match your environment
Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx512, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Processing /home/age87/Liftoff-1.6.3-py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2/pysam-0.20.0+computecanada-cp38-cp38-linux_x86_64.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/networkx-3.0+computecanada-py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/ujson-5.4.0+computecanada-cp38-cp38-linux_x86_64.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic/numpy-1.23.0+computecanada-cp38-cp38-linux_x86_64.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/interlap-0.2.6+computecanada-py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/pyfaidx-0.6.4+computecanada-py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/gffutils-0.11.1+computecanada-py3-none-any.whl
Requirement already satisfied: parasail>=1.2 in /home/age87/MYENV/lib/python3.8/site-packages (from Liftoff==1.6.3) (1.2+computecanada)
Requirement already satisfied: biopython>=1.76 in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx512/MPI/gcc9/openmpi4/busco/5.2.2/lib/python3.8/site-packages (from Liftoff==1.6.3) (1.79)
Requirement already satisfied: six in /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx512/Core/ipykernel/2021a/lib/python3.8/site-packages (from pyfaidx>=0.5.8->Liftoff==1.6.3) (1.16.0)
Requirement already satisfied: setuptools>=0.7 in /home/age87/MYENV/lib/python3.8/site-packages (from pyfaidx>=0.5.8->Liftoff==1.6.3) (46.1.3)
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/argh-0.26.2+computecanada-py2.py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/argcomplete-2.0.0+computecanada-py2.py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/simplejson-3.17.6+computecanada-cp38-cp38-linux_x86_64.whl
Requirement already satisfied: wheel in /home/age87/MYENV/lib/python3.8/site-packages (from parasail>=1.2->Liftoff==1.6.3) (0.34.2)
ERROR: scipy 1.7.0+computecanada has requirement numpy<1.23.0,>=1.16.5, but you'll have numpy 1.23.0+computecanada which is incompatible.
Installing collected packages: pysam, networkx, ujson, numpy, interlap, pyfaidx, argh, argcomplete, simplejson, gffutils, Liftoff
  Attempting uninstall: numpy
    Found existing installation: numpy 1.21.0+computecanada
    Not uninstalling numpy at /cvmfs/soft.computecanada.ca/easybuild/software/2020/avx512/Core/scipy-stack/2021a/lib/python3.8/site-packages, outside environment /home/age87/MYENV
    Can't uninstall 'numpy'. No files were found to uninstall.
Successfully installed Liftoff-1.6.3 argcomplete-2.0.0+computecanada argh-0.26.2+computecanada gffutils-0.11.1+computecanada interlap-0.2.6+computecanada networkx-3.0+computecanada numpy-1.23.0+computecanada pyfaidx-0.6.4+computecanada pysam-0.20.0+computecanada simplejson-3.17.6+computecanada ujson-5.4.0+computecanada

### Error:
$ liftoff
Traceback (most recent call last):
  File "/home/age87/MYENV/bin/liftoff", line 5, in <module>
    from liftoff.run_liftoff import main
  File "/home/age87/MYENV/lib/python3.8/site-packages/liftoff/run_liftoff.py", line 1, in <module>
    from liftoff import write_new_gff, liftover_types, polish, align_features, lift_features
  File "/home/age87/MYENV/lib/python3.8/site-packages/liftoff/polish.py", line 5, in <module>
    import parasail
  File "/home/age87/MYENV/lib/python3.8/site-packages/parasail/__init__.py", line 70, in <module>
    raise Exception("failed to locate and open '{}'".format(_libname))
Exception: failed to locate and open 'libparasail.so'

### Same error with input files
$ liftoff -g GCF_000186305.1_Python_molurus_bivittatus-5.0.2_genomic.gtf Python_molurus_bivittatus_5.0.2_HiC.fa GCF_000186305.1_Python_molurus_bivittatus-5.0.2_genomic.fa
Traceback (most recent call last):
  File "/home/age87/MYENV/bin/liftoff", line 5, in <module>
    from liftoff.run_liftoff import main
  File "/home/age87/MYENV/lib/python3.8/site-packages/liftoff/run_liftoff.py", line 1, in <module>
    from liftoff import write_new_gff, liftover_types, polish, align_features, lift_features
  File "/home/age87/MYENV/lib/python3.8/site-packages/liftoff/polish.py", line 5, in <module>
    import parasail
  File "/home/age87/MYENV/lib/python3.8/site-packages/parasail/__init__.py", line 70, in <module>
    raise Exception("failed to locate and open '{}'".format(_libname))
Exception: failed to locate and open 'libparasail.so'

**FIXED**

Logged off and did:
source ~/liftoff_env
liftoff

**Comment by garfring** [comment]
Fixed it

**Comment by KarenGoncalves** [comment]
Hi, I am having the same issue, but your solution is not working for me. 
I am in using an HPC server which doesn't allow the use of conda.
I created the virtual environment and installed liftoff with "pip install Liftoff" (I did not clone the repo). It installs fine, but when I run liftoff, it raises the exception

`Exception: failed to locate and open 'libparasail.so'`

After resetting the server softwares, reloading the ones I required and reactivating the environment, it continued raising the same exception.
